src/pages/using-spark/guides/writing-meaningful-web-content-with-semantic-html.mdx

```diff
@@ -0,0 +1,250 @@
+
+- [The Strong Element](#the-strong-element)
+- [The Emphasis Element](#the-emphasis-element)
+
+> To view a list of all the HTML elements available to add semantic meaning to copy, refer to [MDN’s Semantic element list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element#inline_text_semantics).
+
+### The Strong Element
+
+The <strong>strong element</strong> indicates strong importance, seriousness, or urgency of the enclosed text inside the `<strong></strong>` tags.
+
+- By default web browsers, render the contents inside strong tags in **bold type**.
+
+#### Usage
+
+Use for:
+
+- Referencing the name of a page
+- Representing strong importance of it's contents
+- Adding importance or urgency to pieces of a sentence
+
+<span id="examples" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h5"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayFive"
+>
+  Examples
+</SprkHeading>
+
+Here are some examples of emphasized word(s) using the strong element.
+
+- Check your **To Do List** items for new tasks.
+  - Check your `<strong> To Do List </strong>` items for new tasks.
+- You can change your communication preferences on **My Profile**.
+  - You can change your communication preferences on `<strong>My Profile</strong>`.
+- **Your payment is due** on September 22, 2021.
+  - `<strong>Your payment is due</strong>` on September 22, 2021.
+- Remember to bring your **driver’s license** to your closing appointment.
+  - Remember to bring your `<strong>driver’s license</strong>` to your closing appointment.
+
+#### The Strong vs The Bold Element vs Styling Text
+
+- Use the **strong element** `<strong></strong>` for important content, and to communicate urgency.
+- Use the <b>bold element</b> `<b></b>` to bring attention to text without indicating that it's more important.
+  - Use for:
+    - Drawing the reader's attention to the elements contents, which aren't of special importance
+    - Keywords in a summary and for product names in a review
+- Use **CSS** (Cascade Styling Sheet) to style or decorate text, like the <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight"> font-weight style property.</a>
+
+<span id="the-emphasis-element" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h3"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayThree"
+>
+  The Emphasis Element
+</SprkHeading>
+
+Use the **emphasis** element to give "stressed emphasis" to the enclosed text inside the `<em></em>` tags, in the same way verbal stress works.
+
+- By default, web browsers render the contents inside the emphasis tags as <i>italic type.</i>
+
+- A screen reader will pronounce the words inside the **emphasis** element `<em></em>` tags with an emphasis, using verbal stress.
+
+> As a UX Writer, is important to know when to apply **stress emphasis** to text, because it can **change the meaning** of the sentence.
```

Comment:
`As a UX Writer, it is important to know when to apply **emphasis** to text because it can **change the meaning** of the sentence.`